benchmark/pom.xml

```diff
@@ -89,6 +89,26 @@
                     <target>1.8</target>
                 </configuration>
             </plugin>
+
+            <plugin>
```

Comment:
consider using `run-benchmarks` profile which is what I believe we use in other projects @alamar @nicktindall ?

Reply:
We do, I think there's a "standard" now in the scripted build config, it runs

`mvn clean install -Dskip.binary -DskipTests`
`mvn test -Prun-benchmarks -Dskip.binary -DuseIncrementalCompilation=false -DskipTests`